src/main/resources/templates/home.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <title>Produkty</title>
+    <link rel="stylesheet" type="text/css" href="/style.css"/>
+</head>
+<body>
+
+<div class="content">
+
+    <a href="/dodaj">Dodaj produkt</a><br/>
+
+    <h2>Produkty:</h2>
+    <a href="/lista">Wszystkie</a><br/>
+    <a href="/lista?kategoria=GROCERIES">Art. spożywcze</a><br/>
```

Comment:
Tutaj jeszcze możesz zastosować mechanizm który przedstawiałem na ostatnich zajęciach z `T(fqn).values()`

Reply:
Nie oglądałem jeszcze filmu nie chciałem się sugerować, ale domyślam się ze chodzi o wyciągnięcie wartości z enuma, np.-spożywcze,

Reply:
tu nie wiem jak do tego podejść w sensie jak wrzucić th: zmienną do linku, tak jak wyżej, chętnie nad tym posiedzę ale mam jeszcze dwa zadania i materiał z baz do przerobienia, oceń jak uważasz a ja jak będę mieć czas to usiądę do tego.